newton/_src/geometry/broad_phase_sap.py

```
@@ -180,7 +196,15 @@ def _sap_project_kernel(
     shape_id = world_index_map[world_slice_start + local_shape_id]
 
     # Project AABB onto direction
-    range = _sap_project_aabb(shape_id, direction, shape_bounding_box_lower, shape_bounding_box_upper, shape_gap)
+    range = _sap_project_aabb(
```

Ruff (0.16.1)

[error] Variable `range` is shadowing a Python builtin

(A001)